querydsl-codegen/src/main/java/com/querydsl/codegen/JavaTypeMappings.java

```diff
@@ -17,6 +17,7 @@
 import com.querydsl.core.types.Expression;
 import com.querydsl.core.types.Path;
 import com.querydsl.core.types.dsl.*;
+import com.querydsl.core.types.dsl.StringTemplate;
```

Comment:
This is added to avoid ambiguous import error because there is StringTemplate class in `java.util.` package